requirements.txt

```diff
@@ -1,4 +1,5 @@
-click>=6,<7
+click>=8
```

Comment:
Lets not bump all the way to 8, I know the minimum requirement for rich click is >=7 so lets stick with that now and bump it later on.